stream/sentinel/src/sentinel.py

```diff
@@ -139,17 +144,17 @@ def _create_indicator(self, data):
                         stix_type = result["attribute"].replace(":value", "")
                         data["type"] = stix_type
                         data["value"] = stix_value
-                        self._create_observable(data)
+                        self._create_observable(data, method)
                     elif result["attribute"] == "file:hashes.'SHA-256'":
                         data["type"] = "file"
                         data["hashes"] = {"SHA-256": stix_value}
-                        self._create_observable(data)
+                        self._create_observable(data, method)
         except:
             self.helper.log_warning(
                 "[CREATE] Cannot convert STIX indicator { " + internal_id + "}"
             )
```

Comment:
Could you realign with the new usage of connector logger?

```suggestion
            self.helper.connector_logger.warning(
                "[CREATE] Cannot convert STIX indicator { " + internal_id + "}"
            )
```